test-data/unit/fine-grained-blockers.test

```diff
@@ -407,3 +407,49 @@ a.py:1: error: invalid syntax
 ==
 b.py:2: error: Incompatible return value type (got "str", expected "int")
 ==
+
+[case testDecodeErrorBlocker-posix]
+import a
+a.f(1)
+[file a.py]
+def f(x: int) -> None: ...
+[file a.py.2]
+# coding: ascii
+ä
+[file a.py.3]
+def f(x: str) -> None: ...
+[out]
+==
+mypy: can't decode file 'tmp/a.py': 'ascii' codec can't decode byte 0xc3 in position 16: ordinal not in range(128)
+==
+main:2: error: Argument 1 to "f" has incompatible type "int"; expected "str"
+
+[case testDecodeErrorBlocker-windows]
```

Comment:
What's the difference between the two tests? Why the platform-specific tags?

Reply:
The offset in the error message is different due to cr/lf.